src/server/controllers/init.js

```diff
@@ -0,0 +1,275 @@
+const axios = require('axios');
+
+const {
+    EXTERNAL_API,
+    CATEGORIES,
+    SERVER_ERROR_MESSAGE,
+    NUMBER_OF_COMMENTS_WITH_PARENT_TO_GENERATE,
+    NUMBER_OF_COMMENTS_TO_GENERATE,
+    NUMBER_OF_POSTS_TO_GENERATE,
+    NUMBER_OF_USERS_AND_PROFILES_TO_GENERATE,
+} = require('../config');
+
+const {
+    fakeUser,
+    fakeProfile,
+    fakePost,
+    fakeComment,
+    fakeCommentWithParent,
+} = require('../utils/faker');
+
+const { prisma } = require('../utils/prisma');
+
+const { capitalizeFirstLetter, generateRandomInt } = require('../utils/utils');
+
+const { getPokemonById } = require('./pokemon');
+
+const initPokemonDatabase = async (req, res) => {
+    const numberOfPokemonToFetch = 151;
+
+    for (let i = 0; i < numberOfPokemonToFetch; i++) {
+        const pokemonId = i + 1;
+
+        const response = await axios(`${EXTERNAL_API}${pokemonId}`);
+        const fetchedPokemon = response.data;
+
+        const filteredPokemon = await filterPokemonData(fetchedPokemon,getPokemonById);
+
+        const createdPokemon = await createNewPokemon(filteredPokemon);
+        console.log('Created Pokemon:', createdPokemon);
+    }
+
+    res.status(201).json('151 Pokemon seeded successfully');
+};
+
+const filterPokemonData = async (pokemon, pokemonId) => {
+    const pokemonName = capitalizeFirstLetter(pokemon.name);
+
+    const types = [];
+    pokemon.types.forEach((type) =>
+        types.push(capitalizeFirstLetter(type.type.name))
+    );
+
+    const pokedexId = pokemonId.toString().padStart(3, '0');
+    const largeImageUrl = `https://assets.pokemon.com/assets/cms2/img/pokedex/full/${pokedexId}.png`;
+    const smallImageUrl = `https://assets.pokemon.com/assets/cms2/img/pokedex/detail/${pokedexId}.png`;
+
+    const filteredPokemon = {
+        name: pokemonName,
+        number: pokemonId,
+        largeImageUrl: largeImageUrl,
+        smallImageUrl: smallImageUrl,
+        baseHP: pokemon.stats[0].base_stat,
+        baseAttack: pokemon.stats[1].base_stat,
+        baseDefense: pokemon.stats[2].base_stat,
+        specialAttack: pokemon.stats[3].base_stat,
+        specialDefense: pokemon.stats[4].base_stat,
+        speed: pokemon.stats[5].base_stat,
+    };
+
+    return (pokemonToCreate = {
+        filteredPokemon,
+        types,
+    });
```

Comment:
This is funky, what's the reasoning behind this? Are you expecting something different with this instead of
```js
return {
  filteredPokemon,
  types
}
```

Reply:
I'm not sure why I decided to assign a constant to that object, I will change to match your suggestion